core/schema/container.go

```diff
@@ -374,6 +375,14 @@ func (s *containerSchema) withoutMount(ctx *router.Context, parent *core.Contain
 	return parent.WithoutMount(ctx, args.Path)
 }
 
+func (s *containerSchema) withMountedSSH(ctx *router.Context, parent *core.Container, _ any) (*core.Container, error) {
+	env := containerWithVariableArgs{
+		Name:  "SSH_AUTH_SOCK",
+		Value: "/ssh-agent.sock",
```

Comment:
Is there a way that this value can be wrong on some hosts? I'm not an expert in ssh but is it possible that the socket's path is different?

Reply:
It's something we mount in the engine, it will always be present